src/test/groovy/com/jmeter/websocket/plugin/endpoint/jetty/JettySessionManagerSpec.groovy

```diff
@@ -0,0 +1,95 @@
+package com.jmeter.websocket.plugin.endpoint.jetty
+
+import org.eclipse.jetty.websocket.api.Session
+import spock.lang.Specification
+import spock.lang.Subject
+import spock.lang.Unroll
+
+@Unroll
+class JettySessionManagerSpec extends Specification {
+
+    @Subject
+    JettySessionManager manager = new JettySessionManager()
+
+    def "Should return false when zero session is registered on hasOpenSession"() {
+        expect:
+        !manager.hasOpenSession(URI.create('ws://localhost:8080/ws'))
+    }
+
+    def "Should return '#value' if '#registeredUri' session is registered and it isOpen()='#isOpen' on hasOpenSession('#uri')"() {
+        given:
+        manager.registerSession(registeredUri,
+                Stub(Session) {
```

Comment:
It looks like you use this stub in many test maybe it is better to make it as shared field or field. Be honest creating such instance looks bad here (((